### ISSUE TITLE
mermaid from imported markdown files

### ISSUE THREAD

**Issue by azatoth** [kamiazya/typedoc-plugin-mermaid#473]
**Describe the bug**
When importing a file into typedoc using `[[include:path/to/file.md]]`, mermaid tags will not be handled and it will complain it can't highlight it.

**To Reproduce**

1. Create typescript `foo.ts`:
```typescript
/**
 * [[include:foo.md]]
 */
export class Foo {

}
```

2. Create `foo.md` with:
````markdown
# Foo

```mermaid
  flowChart TB
    A --> B
```
````
3. Execute typedoc:
typedoc src/foo.ts --logLevel Verbose --includes src/

> Info: Loaded plugin typedoc-plugin-mermaid
> Debug: Using TypeScript 4.3.5 from ./node_modules/typescript/lib
> Debug: Converting with 1 programs 1 entry points
> Debug: Finished getting entry points in 7598ms
> Debug: Begin readme search at ./src
> Debug: Finished conversion in 99ms
> Debug: Renderer: Loading highlighter took 472ms
> Warning: 
> Unsupported highlight language "mermaid" will not be highlighted. Run typedoc --help for a list of supported languages.
> target code block :
> 	  flowChart TB
> 	    A --> B
> source files :undefined
> output file :
> 	undefined
> Info: Documentation generated at ./docs
> Debug: HTML rendering took 493ms

**Expected behavior**
I expect mermaid blocks be rendered from the included markdown file

**Comment by kamiazya** [comment]
@azatoth 
It didn't seem to work because the Markdown conversion and file loading timings didn't match.

PR Welcome.

**Comment by kamiazya** [comment]
fixed by #560

Thanks for @WHenderson !
